docs/installing-solidity.rst

```diff
@@ -32,102 +32,119 @@ Details about the usage of the Node.js package can be found in the
 Binary Packages
```

Comment:
The comment about using browser-solidity without internet is outdated (you cannot just save the page anymore). We can change it to:
`go to https://github.com/ethereum/browser-solidity/tree/gh-pages and download the .zip file as explained on that page.`